packages/ref-metrics-ilamb/tests/unit/test_metrics.py

```diff
@@ -94,3 +95,50 @@ def test_standard_metric(tmp_path, cmip6_data_catalog):
     assert result.successful
     assert output_bundle_path.exists()
     assert output_bundle_path.is_file()
+
+
+def test_standard_grid(tmp_path, cmip6_data_catalog):
+    metric = ILAMBStandard(
+        registry_file="test.txt", sources={"gpp": "test/Grid/gpp.nc"}, relationships={"pr": "test/Grid/pr.nc"}
+    )
+    grp = cmip6_data_catalog[
+        (cmip6_data_catalog["experiment_id"] == "historical")
+        & ((cmip6_data_catalog["variable_id"] == "gpp") | (cmip6_data_catalog["variable_id"] == "pr"))
+    ].groupby(["source_id", "member_id", "grid_label"])
+    _, ds = next(iter(grp))
+    output_directory = tmp_path / "output"
+    (output_directory / metric.slug).mkdir(parents=True, exist_ok=True)
+
+    definition = MetricExecutionDefinition(
+        output_directory=output_directory,
+        output_fragment=pathlib.Path(metric.slug),
+        key="ilamb-standard-test_test",
+        metric_dataset=MetricDataset(
+            {
+                SourceDatasetType.CMIP6: DatasetCollection(ds, "instance_id"),
+            }
+        ),
+    )
+
+    result = metric.run(definition)
+
+    assert str(result.bundle_filename) == "output.json"
+
+    output_bundle_path = definition.output_directory / definition.output_fragment / result.bundle_filename
+
+    assert result.successful
+    assert output_bundle_path.exists()
+    assert output_bundle_path.is_file()
+
+
+def test_standard_fail():
+    try:
+        ILAMBStandard(registry_file="test.txt", sources={"gpp": "test/Grid/gpp.nc", "pr": "test/Grid/pr.nc"})
+        assert False
+    except ValueError:
+        pass
```

Comment:
```suggestion
    with pytest.raises(ValueError):
        ILAMBStandard(registry_file="test.txt", sources={"gpp": "test/Grid/gpp.nc", "pr": "test/Grid/pr.nc"})
```
Use pytest.raises to check that a given exception is raised. You can also assert on the exception message if you need to

Reply:
Thanks for the pointer, I am not terribly well versed in pytest.